toolkit/scripts/setuplkgtoolchain.sh

```diff
@@ -6,13 +6,14 @@ set -e
 
 ROOT_FOLDER=$(git rev-parse --show-toplevel)
 LKG_FILENAME="lkg-3.0-dev.json"
+PROCESSOR=$(uname -m)
```

Comment:
nit: consider renaming to machine because -p is for processor, and -m for machine, or was this intentional?

Reply:
+1, I'd suggest actually calling the variable ARCHITECTURE

Reply:
updated